package.json

```diff
@@ -70,7 +70,8 @@
   },
   "files": [
     "dist",
-    "bin"
+    "bin",
+    "blog_example"
```

Comment:
I do not want to include blog_example. Instead, can you create an init directory and in there put in the files that we will want to start with?